client/chainlink_models.go

```diff
@@ -1,25 +1,33 @@
 package client
```

Comment:
Please, also move `basic_http_client.go` and remove `client_suite.go`. We won't need them after refactoring with `resty`.

Reply:
`basic_http_client.go` is being used by other clients like kafka, mockserver. I can remove `client_suite.go`